lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -459,6 +460,15 @@ def non_fractional_currency?(currency)
       def emv_payment?(payment)
         payment.respond_to?(:emv?) && payment.emv?
       end
+
+      def error_code_from(response)
+        code = response['error']['code'] 
+        decline_code = response['error']['decline_code'] if code == 'card_declined'
```

Comment:
You can use `STANDARD_ERROR_CODE[:card_declined]` here instead of a magic string